app/src/main/java/com/cornellappdev/uplift/ui/screens/ProfileCreationScreen.kt

```diff
@@ -124,23 +132,35 @@ fun ProfileCreationScreen() {
             Spacer(modifier = Modifier.height(25.dp))
 
             Column(verticalArrangement = Arrangement.spacedBy(8.dp)) {
-                InfoCheckboxRow(checkedState, checkboxColors, "I agree with EULA terms and agreements")
+                InfoCheckboxRow(
+                    checkedState,
+                    checkboxColors,
+                    "I agree with EULA terms and agreements"
+                )
 
-                InfoCheckboxRow(checkedState2, checkboxColors, "I allow Uplift to access data on my gym usage")
+                InfoCheckboxRow(
+                    checkedState2,
+                    checkboxColors,
+                    "I allow Uplift to access data on my gym usage"
+                )
 
-                InfoCheckboxRow(checkedState3, checkboxColors, "I allow Uplift to access my location")
+                InfoCheckboxRow(
+                    checkedState3,
+                    checkboxColors,
+                    "I allow Uplift to access my location"
+                )
```

Comment:
Small nitpick, let's make these named arguments for readability (i.e. `paramName = arg`)